Source/ACE.Server/Config.js.docker

```diff
@@ -182,6 +182,18 @@
 
     // After updating to latest world database, automatically import further customizations
     // AutoUpdateWorldDatabase must be true for this option to be used
-    "AutoApplyWorldCustomizations": true
+    // SQL files will be executed given the sort order of the full paths of the files
+    "AutoApplyWorldCustomizations": true,
+
+    // When AutoApplyWorldCustomizations is set to true, the auto apply process will search for
+    // all .sql files in the following directories.
+    // This process will still use ./Content by default, or the the config_properties_string
+    // value for 'content_folder' if it exists
+    // Example: [ "C:\MyContent", "C:\FTPRoot\Editor1Content", "C:\FTPRoot\Editor2Content" ]
```

Comment:
```suggestion
    // Example: [ "C:\\MyContent", "C:\\FTPRoot\\Editor1Content", "C:\\FTPRoot\\Editor2Content" ]
```paths seem to be safer with `\\` instead of `\`